Markup/Editor/WelcomePage.html

```diff
@@ -0,0 +1,268 @@
+<!DOCTYPE html>
+<!--
+~   Copyright (c) WSO2 Inc. (http://wso2.com) All Rights Reserved.
+~
+~   Licensed under the Apache License, Version 2.0 (the "License");
+~   you may not use this file except in compliance with the License.
+~   You may obtain a copy of the License at
+~
+~        http://www.apache.org/licenses/LICENSE-2.0
+~
+~   Unless required by applicable law or agreed to in writing, software
+~   distributed under the License is distributed on an "AS IS" BASIS,
+~   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+~   See the License for the specific language governing permissions and
+~   limitations under the License.
+-->
+<html lang="en">
+  <head>
+    <meta charset="utf-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <!-- The above 3 meta tags *must* come first in the head; any other head content must come *after* these tags -->
+
+    <title>Stream Processor Studio</title>
+    <link href="libs/theme-wso2-2.0.0/images/favicon.png" rel="icon" type="image/png" sizes="16x16" />
+    
+    <!-- WSO2 Theme -->
+    <link href="libs/theme-wso2-2.0.0/css/theme-wso2.css" rel="stylesheet" type="text/css" />
+    
+    <!-- Font WSO2 CSS -->
+    <link href="libs/font-wso2-1.2.2/css/font-wso2.min.css" rel="stylesheet" type="text/css" />
+    
+    <link href="css/welcome-screen.css" rel="stylesheet" type="text/css" />
+
+    <!-- HTML5 shim and Respond.js for IE8 support of HTML5 elements and media queries -->
+    <!-- WARNING: Respond.js doesn't work if you view the page via file:// -->
+    <!--[if lt IE 9]>
+    <script src="libs/html5shiv_3.7.2/html5shiv.min.js"></script>
+    <script src="libs/respond_1.4.2/respond.min.js"></script>
+    <![endif]-->
+
+    <style type="text/css">
```

Comment:
Created an issue [1], I think we need to have this class. This is useful. 

[1] https://github.com/wso2-dev-ux/theme-wso2/issues/53